src/store.ts

```diff
@@ -5,6 +5,7 @@ import type { MutableRefObject } from 'react'
 import type { WorkerApi } from '@react-three/cannon'
 import type { GetState, SetState, StateSelector } from 'zustand'
 import type { Session } from '@supabase/supabase-js'
+import { Mesh } from 'three'
```

Comment:
if it's being used as a type, we should specify that in the import. I wonder if there is a lint rule for that.

```suggestion
import type { Mesh } from 'three'
```

Reply:
That's why I actually got a ESLint PR open 😄